install.sh

```diff
@@ -237,8 +223,7 @@ function download_check () {
     # Use cached file if available and caching is enabled.
     if [ -n "$CREW_CACHE_ENABLED" ] && [[ -f "$CREW_CACHE_DIR/${3}" ]] ; then
       mkdir -p "$CREW_CACHE_DIR"
-      # sudo chown -R "$(id -u)":"$(id -g)" "$CREW_CACHE_DIR" || true
-      chown -R "$(id -u)":"$(id -g)" "$CREW_CACHE_DIR" || true
+      sudo chown -R "$(id -u)":"$(id -g)" "$CREW_CACHE_DIR" || true
```

Comment:
Why not just use `chronos` verbatim since we are requiring a login with this user anyway?

Reply:
Possibly because the system may have a custom configuration, or other users?

To be honest I'm not entirely sure, but it doesn't look like we've used chronos verbatim for 7 years (if my git grep skills are to be trusted, which is not a given).

At any rate, its a total difference of 6 characters, and keeping it as is may help some obscure setups (custom user on ChromiumOS, maybe).

Reply:
I prefer not to hardcoding it also.

Reply:
How about a compromise?  
```
: "${CREW_USER:=chronos}"
sudo chown -R "$CREW_USER":"$CREW_USER" "$CREW_CACHE_DIR" || true
```

Reply:
This seems like it would go against the user's expectations, as if they are not the user `chronos` they will not be able to use crew without setting this variable.

Reply:
The VT-2 session requires the user to login as chronos.  Unless they create a new user, there is no reason not to require chronos to login.

Reply:
And if they have created a new user, the install will be broken by default.

Reply:
That's what I'm saying.  Just require chronos user.

Reply:
Why? What's wrong with supporting other users?

Reply:
You cannot login to VT-2 and use sudo unless you are chronos.  If somebody creates a new user, they will need to add to sudo users or the install will fail.

Reply:
True, but if the new user does have sudo perms then there's no reason for the install to arbitrarily fail, and if the new user does not have sudo perms then the install will fail before that. 

And besides, we still have users who are pre M117 and do not need the VT-2 workaround.

Reply:
I think the VT-2 workaround should be the standard installation instructions for all users.  It works on older releases too and most of the users will upgrade to the latest ChromeOS milestone anyway (as long as their hardware is not EOL).  It would be nice to autologin to a VT-2 session.  Maybe @supechicken can figure out how to do it. :smiley: